code/controllers/subsystem/fire_burning.dm

```diff
@@ -0,0 +1,37 @@
+SUBSYSTEM_DEF(fire_burning)
+	name = "Fire Burning"
```

Comment:
This would look a lot better in the code as just SSfire, honestly

Reply:
Isn't that supposed to be the standard naming convention for it anyway?

Reply:
I'd rather keep consistency here, especially since every single other subsystem, thus far, is in line with this (and those to be ported).

Reply:
Make a /tg/ PR too then :stuck_out_tongue: 

I'm not really worried about parity with fire or the atmos controller until we port listmos and /tg/'s pipes entirely.

Reply:
I rather have [these kind of pipes](https://i.gyazo.com/7a51e72e935f73a5d76cafd075a9bca6.png)